pkg/didcomm/crypto/crypter.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+// Crypter is an Aries envelope encrypter to support
+// secure DIDComm exchange of envelopes between Aries agents
+type Crypter interface {
+	// Encrypt a payload in an Aries compliant format
+	// returns:
+	// 		[]byte containing the encrypted envelope
+	//		error if encryption failed
+	Encrypt(payload string) ([]byte, error)
```

Comment:
I think payload is better represented as []byte.

See https://github.com/square/go-jose/blob/8bad6148bd0a8da57f9840693f6f611ac4483d09/crypter.go#L31

Reply:
will update